### PR TITLE
Update Components to SP7

### LINKED ISSUE #197: Update components to match SP7 Alpha draft
### Description

First pass getting the visual components into a final state for our Alpha content release. [Reference conversation & figma](https://nava.slack.com/archives/C01Q5640YFR/p1622046332013600)

This work should include an ~hour pairing with Design at the 90% point to make final calls on which details are worth pushing to 100% and which are fungible. 

Existing S6 | S7 Alpha 3 
--|--
image | image

### Acceptance Criteria

  - [x] Change Feedback component to an above-header alert banner
  - [x] Remove Breadcrumbs
  - [x] Remove `Manage Claim` Button
  - [x] Proper Heading cadence
    - H1 `Claimtracker`
    - H2 `Claim Status` & `Claim Details`
    - H3 `Next Steps` 
  - [x] Proper vertical spacing between elements
  - [x] Paired with design on final 10% styling efforts

### PR DESCRIPTION
**Pull request by lomky**
Updates to match the site design as of SP7, based on the presentation view on Figma.

Mode | Before | After
--|--|--
Desktop | image | image
Mobile | image | image

Resolves #197 

  - [x] Change Feedback component to an above-header alert banner
  - [x] Remove Breadcrumbs
  - [x] Remove `Manage Claim` Button
  - [x] Proper Heading cadence
    - H1 `Claimtracker`
    - H2 `Claim Status` & `Claim Details`
    - H3 `Next Steps` 
  - [x] Proper vertical spacing between elements
  - [x] Paired with design on final 10% styling efforts
  - [ ] Changes are tested on Staging post-merge into `main`

### CODE REVIEW

File: .eslintrc.js
Comment (lomky) [inline .eslintrc.js:44]: some components began throwing the default alert due to not understanding typescript fully - the TS one fully understands how to trace it.

File: components/ClaimDetails.tsx
Comment (lomky) [inline components/ClaimDetails.tsx:37]: Moved to be in the first col instead of its own row.

File: components/ClaimStatus.tsx
Comment (lomky) [inline components/ClaimStatus.tsx:27]: Corrected to a third level heading

File: components/ClaimStatus.tsx
Comment (lomky) [inline components/ClaimStatus.tsx:40]: Updated Date & Button both removed.

File: components/Header.tsx
Comment (lomky) [inline components/Header.tsx:12]: Definitely `Hide Whitespace Changes` for this file! Adding a container makes this behave responsively like the Footer does.

File: components/InfoField.tsx
Comment (lomky) [inline components/InfoField.tsx:4]: No longer used

File: components/Main.tsx
Comment (lomky) [inline components/Main.tsx:23]: Removed Components (new component goes into index.tsx)

File: components/TimeoutModal.tsx
Comment (lomky) [inline components/TimeoutModal.tsx:42]: Reordered unchanged based on linter warning

File: components/TimeoutModal.tsx
Comment (lomky) [inline components/TimeoutModal.tsx:68]: Reordered unchanged based on linter warning

File: package.json
Comment (lomky) [inline package.json:13]: Corrects static images on local dev

File: styles/globals.scss
Comment (lomky) [inline styles/globals.scss:7]: Lots of styling tweaks, sorry

Comment (lomky) [reply styles/globals.scss:7]: I had been mixing up em/rem - rem is the one that is set through the page as compared to em which is set by the parent element. we want rem

File: styles/globals.scss
Comment (lomky) [inline styles/globals.scss:153]: Keep reasonable padding regardless of screen size

File: tests/pages/index.test.tsx
Comment (lomky) [inline tests/pages/index.test.tsx:27]: this was looking at a breadcrumb, which is now defunct. Look for the h1 string instead.

File: styles/globals.scss
Comment (nguyen-tina) [reply styles/globals.scss:7]: To confirm, we want rem so that the page can be proportional to whatever the browser determines is the "baseline" spacing?

**Review by nguyen-tina — COMMENTED** [review]
This is looking good to me! 

Nit: For the left padding spacing for h3 Next Steps, could we change that to 1rem instead of 1.5rem?

**Comment by lomky** [comment]
@nguyen-tina can do! should that also affect the bullets below?

**Comment by nguyen-tina** [comment]
No, I don't think so! @lomky

File: components/WorkInProgress.tsx
Comment (rocketnova) [inline components/WorkInProgress.tsx:25]: React learning question: Why do we have text here and also have text in public/locales/en/common.json?

**Review by rocketnova — APPROVED** [review]
LGTM, but you probably want to wait for Kalvin to also do a review. I didn't look super closely at the tests.

File: components/WorkInProgress.tsx
Comment (lomky) [reply components/WorkInProgress.tsx:25]: Its how the Trans component works! https://react.i18next.com/latest/trans-component

it does some parsing of the default language so that you can reference the links in other languages

**Review by kalvinwang — APPROVED** [review]
lgtm, thanks! also there's this warning which doesn't need to be fixed in this PR, but it looks like it came from your shimmer PR so just mentioning it here

image

File: components/WorkInProgress.tsx
Comment (lomky) [reply components/WorkInProgress.tsx:25]: its got strong ReGex Magic vibes, tbh.

File: styles/globals.scss
Comment (lomky) [reply styles/globals.scss:139]:
```suggestion
  //override bootstrap for our NextSteps component layout
```

File: styles/globals.scss
Comment (lomky) [reply styles/globals.scss:153]:
```suggestion
    //Keep reasonable padding regardless of screen size
    .main-content {
      padding: 2rem 1rem;
    }
```

File: utils/queryApiGateway.tsx
Comment (lomky) [inline utils/queryApiGateway.tsx:23]: ESLint made me move these after the merge conflict touched the file. They are moved unchanged.

**Pull request merged**
